Source/DataSources/KmlDataSource.js

```diff
@@ -1236,7 +1237,8 @@ define([
             oneTimeWarning('kml-gx:drawOrder', 'KML - gx:drawOrder is not supported in LineStrings');
         }
 
-        var coordinates = readCoordinates(coordinatesNode);
+        var ellipsoid = defaultValue(dataSource._ellipsoid, Ellipsoid.WGS84);
```

Comment:
Can we set this default value on the data source once before all these functions are called?